discovery-provider/alembic/trigger_sql/ddl.sql

```diff
@@ -36,4 +36,25 @@ commit;
 -- 4/4/23: drop blocks_copy
 begin;
   drop table if exists blocks_copy;
+commit;
+
+-- 4/10/23: inverse supporter rank up and supporting rank up notifications
+-- context: supporter rank up and supporting rank up notifications were
+-- reversed when we first added them to the handle_supporter_rank_up.sql trigger.
+-- this migration is meant to swap those types for any notification made when this migration
+-- was first run. we indicate whether the entry is from when a migration was first run
+-- by looking at whether type_v2 is null. supporter/ing_rank_up notifications
+-- created after this migration was run should have type_v2 as not null
+-- (see handle_supporter_rank_ups.sql)
+begin;
+alter table notification
```

Comment:
nit we should probably settle on UPPERCASE or lowercase for the sql in this file.
curious what your take is on it though after having gone through this exercise!

Reply:
i feel like upperclase is cleaner but lowercase is probably easier to get consistently, so i will switch to lowercase